lib/hypertypes/hypertypes/ERC4626HyperdriveDeployerContract.py

```diff
@@ -45,7 +52,13 @@ def __init__(self, address: ChecksumAddress | None = None, abi=Any) -> None:
         self.abi = abi
         # TODO: make this better, shouldn't initialize to the zero address, but the Contract's init
         # function requires an address.
-        self.address = address if address else cast(ChecksumAddress, "0x0000000000000000000000000000000000000000")
+        self.address = (
+            address
+            if address
+            else cast(
+                ChecksumAddress, "0x0000000000000000000000000000000000000000"
+            )
+        )
```

Comment:
```suggestion
        self.address = (address or cast(
                ChecksumAddress, "0x0000000000000000000000000000000000000000"
            )
        )
```

Reply:
this would also replace "falsy" values like an empty string 🤔

Reply:
wait, this is machine generated isn't it

Reply:
yeah it is, but we can change the generated code.  would be a suggestion for pypechain though, which i can do.